packages/camera-web/test/SimpleCameraHUD/SimpleCameraHUD.test.tsx

```diff
@@ -38,51 +40,33 @@ describe('SimpleCameraHUD component', () => {
     unmount();
   });
 
-  it('should display a take picture button that calls the handle.takePicture method', () => {
-    const handle = { takePicture: jest.fn() };
-    const { unmount } = render(<SimpleCameraHUD handle={handle} />);
-
-    fireEvent.click(screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID));
-    expect(handle.takePicture).toHaveBeenCalled();
-    unmount();
-  });
-
-  it('should not throw any error when no handle is provided', () => {
-    expect(() => {
-      const { unmount, rerender } = render(<SimpleCameraHUD />);
-      fireEvent.click(screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID));
-
-      rerender(<SimpleCameraHUD handle={{}} />);
-      fireEvent.click(screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID));
-      unmount();
-    }).not.toThrow();
-  });
-
   it('should disable the take picture button if the camera is loading', () => {
     const { unmount, rerender } = render(<SimpleCameraHUD />);
-
-    let takePictureBtn = screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID);
-    expect((takePictureBtn as HTMLButtonElement).disabled).toBe(false);
+    expectPropsOnChildMock(TakePictureButtonMock, {
+      disabled: false,
+      onClick: undefined,
+      size: 60,
+    });
 
     rerender(<SimpleCameraHUD handle={{ isLoading: true }} />);
-    takePictureBtn = screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID);
-    expect((takePictureBtn as HTMLButtonElement).disabled).toBe(true);
+    expectPropsOnChildMock(TakePictureButtonMock, { disabled: true, onClick: undefined, size: 60 });
     unmount();
```

Comment:
In this test, you only want to check if the button is disabled, you don't care about other props. That means that you should ONLY check for the disabled prop. That way, if for some reason the other props (for example the size prop) are not passed properly, this test will not fail (because this test only checks for the disabled prop). For this, you can do something like this :

```typescript jsx
it('should disable the take picture button if the camera is loading', () => {
  const { unmount, rerender } = render(<SimpleCameraHUD />);
  expectPropsOnChildMock(TakePictureButtonMock, { disabled: false });

  rerender(<SimpleCameraHUD handle={{ isLoading: true }} />);
  expectPropsOnChildMock(TakePictureButtonMock,  { disabled: true });
  unmount();
});
```